src/chi/components/picker/picker.scss

```diff
@@ -14,6 +14,124 @@ $sizes: (
   )
 );
 
+.m-picker {
+  input {
+    &[type='checkbox'],
+    &[type='radio'] {
+      &.a-picker {
+        & + label {
+          span.a-picker__checkbox {
+            border: 2px solid set-color(grey, 50);
+            border-radius: 3px;
+            color: set-color(grey, 50);
+            height: 1rem;
+            position: relative;
+            width: 1rem;
+
+            &::before {
```

Comment:
Change to:
```
              border: 2px solid set-color(grey, 50);
              border-right: none;
              border-top: none;
              content: '';
              height: 0.3125rem;
              left: 0.0625rem;
              position: absolute;
              top: 0.125rem;
              transform: rotate(-45deg) scale(0);
              transition-duration: 0.2s;
              width: 0.625rem;
```

This checkmark will look more consistent with our checkmark icon. For the left and top properties, i'm not sure if the calc is necessary given that the checkbox will always render at 1x1rem. If it were to render larger, we would need to adjust other values to support that new size.